doc/2/api/controllers/security/update-user/index.md

```diff
@@ -68,6 +68,10 @@ Returns the update user kuid and version number.
   "requestId": "<unique request identifier>",
   "result": {
     "_id": "<kuid>",
+    "_source": {
+      "profileIds": ["<profileId>"],
```

Comment:
Nitpicking: since you are giving an example fullname below you can give an example profile here instead of a placeholder

Reply:
Let's keep consistency then :+1: